Dockerfile.frontend

```diff
@@ -0,0 +1,38 @@
+FROM --platform=linux/amd64 ubuntu@sha256:bbf3d1baa208b7649d1d0264ef7d522e1dc0deeeaaf6085bf8e4618867f03494 AS deps
+# Note: The above is ubuntu 22.04
+
+# Install required tools
+RUN DEBIAN_FRONTEND=noninteractive apt update && apt install -y \
+    curl \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install node
+RUN curl --fail -sSf https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
+ENV NVM_DIR=/root/.nvm
+COPY .nvmrc .node-version
+RUN cat .node-version
+RUN . "$NVM_DIR/nvm.sh" && nvm install "$(cat .node-version)"
```

Comment:
```
nvm install
```
will pick up the version in the `.nvmrc` and set it to the current version, so you can simplify the line above and probably delete the line below.